data/net/minecraft/core/Vec3i.mapping

```diff
@@ -10,14 +11,14 @@ CLASS net/minecraft/core/Vec3i
 		ARG 2 y
 		ARG 3 z
 	METHOD above ()Lnet/minecraft/core/Vec3i;
-		COMMENT Offset this vector 1 block up
+		COMMENT Creates a new Vec3i one unit up.
 	METHOD above (I)Lnet/minecraft/core/Vec3i;
-		COMMENT Offset this vector upwards by the given distance.
+		COMMENT Creates a new Vec3i up by the distance.
 		ARG 1 distance
 	METHOD below ()Lnet/minecraft/core/Vec3i;
-		COMMENT Offset this vector 1 block down
+		COMMENT Creates a new Vec3i one unit down.
 	METHOD below (I)Lnet/minecraft/core/Vec3i;
-		COMMENT Offset this vector downwards by the given distance.
+		COMMENT Creates a new Vec3i down by the distance.
```

Comment:
This is technically incorrect, since if the distance is `0`, then it returns the same vector object.